app/helpers/application_helper.rb

```diff
@@ -7,4 +7,12 @@ def bootstrap_flash_class(level)
     when 'alert' then "alert alert-warning"
     end
   end
+
+    def shorten_description(string, max_length = 280)
+    if string.length > max_length
+      string[0, max_length - 3] + "..."
```

Comment:
yeah but this kind of methods are a great examples when objects should be decorated. Please take a look at https://github.com/drapergem/draper gem and introduce it in meecon app. Then create an event decorator class and use it :)